### PR TITLE
RaspberryPi 4 (64bit) support

### PR DESCRIPTION
**Pull request by agherzan**

### CODE REVIEW

**Comment by floion** [comment]
@resin-jenkins add to whitelist

**Comment by floion** [comment]
@agherzan this PR needs rebasing

**Comment by agherzan** [comment]
@floion rebased and updated icon.

File: raspberrypi4-64.coffee [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	slug: 'raspberrypi4-64'
+	aliases: [ 'raspberrypi4-64' ]
+	name: 'Raspberry Pi 4 (using 64bit OS)'
+	arch: 'aarch64'
+	state: 'experimental'
+
+	instructions: commonImg.instructions
+	gettingStartedLink:
+		windows: 'https://docs.resin.io/raspberrypi3/nodejs/getting-started/#adding-your-first-device'
+		osx: 'https://docs.resin.io/raspberrypi3/nodejs/getting-started/#adding-your-first-device'
```
Comment (floion) [inline raspberrypi4-64.coffee]: This should point to rpi4 docs section, even if not existing now, rather than point to rpi3 docs

**Comment by floion** [comment]
@balena-ci retest

File: raspberrypi4-64.coffee (hunk 1 above)
Comment (agherzan) [reply raspberrypi4-64.coffee]: Fixed.

File: layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi-4.19/d9aa3902c1fb7163edf1812dd0e03ed5a7e129ed.patch [hunk 2]
```diff
@@ -0,0 +1,69 @@
+From d9aa3902c1fb7163edf1812dd0e03ed5a7e129ed Mon Sep 17 00:00:00 2001
```
[the remark this answers is not in the document]
Comment (agherzan) [reply layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi-4.19/d9aa3902c1fb7163edf1812dd0e03ed5a7e129ed.patch]: No idea: https://github.com/balena-os/balena-raspberrypi/issues/344

**Comment by agherzan** [comment]
@balena-ci retest

**Comment by agherzan** [comment]
@floion please re-review.

**Comment by agherzan** [comment]
Don't merge yet. It needs switch to overlayfs.

**Comment by agherzan** [comment]
Switched to overlayfs. Ready to merge.

**Comment by agherzan** [comment]
@balena-ci retest

**Comment by floion** [comment]
@balena-ci retest

**Comment by floion** [comment]
@balena-ci retest

**Comment by floion** [comment]
@resin-jenkins test this please

**Comment by floion** [comment]
@balena-ci retest

**Comment by floion** [comment]
@balena-ci retest

**Pull request merged**